convert2rhel/actions/__init__.py

```diff
@@ -59,11 +60,12 @@
 #:      tests can be run.
 #:
 #: .. warning::
-#:      * Do not change the numeric value of these statuses once they
+#:      * Do not change the numeric value of these levels once they
 #:        have been in a public release as external tools may be depending on
 #:        the value.
-#:      * Actions should not set a status to ``SKIP``.  The code which
+#:      * Actions should not set a level to ``SKIP``.  The code which
 #:        runs the Actions will set this.
+# add info status code as 25
```

Comment:
This will be done as part of this PR?

Reply:
Not as part of this PR, once we finish the PR to evaluate if we have any info messages to add or not we will decide whether we need to add it or not